### ISSUE TITLE
Expokit barcode scanner error

### ISSUE THREAD

**Issue by HJ29** [expo/expo#1967]
### Environment

expokit: 2.6.5
IOS version: 9.0
react-native: https://github.com/expo/react-native/archive/sdk-27.0.0.tar.gz, version: 0.55.2

### Steps to Reproduce

1. Just use the demo code from expo website.
![screen shot 2018-07-09 at 10 49 14 pm](https://user-images.githubusercontent.com/24311533/42457561-5b01ebf2-83ca-11e8-8784-784ff2b61b3e.png)

### Expected Behavior

1. It is expected to work well. It worked on my android code but not for IOS.

### Actual Behavior

1. Prompt error
![screen shot 2018-07-09 at 10 35 41 pm](https://user-images.githubusercontent.com/24311533/42457468-223e0800-83ca-11e8-9905-c2fee381f640.png)

2. I am able to solve it by adding this two line in camera code in node modules:
    pictureSize: true,
    onPictureSaved: true,

So I am wondering is it a bug or what I had missed.
![screen shot 2018-07-09 at 10 48 19 pm](https://user-images.githubusercontent.com/24311533/42457510-3cc551d8-83ca-11e8-8f45-62d6b9742eb3.png)

**Comment by terribleben** [comment]
Hi! It looks like your project is on SDK 27 but your native code is using SDK 28. The `pictureSize` prop was added in SDK 28, but your SDK 27 JS doesn't know about it, so it's throwing an error. That's what the red screen meant when it said "this usually means that your versions of the native code and JavaScript code are out of sync."

If you'd like to keep your project on SDK 27, I recommend  switching your ExpoKit tag to `2.5.x` and re-running `pod install`.

Let me know if that works for you.

**Comment by HJ29** [comment]
![screen shot 2018-07-10 at 4 34 35 pm](https://user-images.githubusercontent.com/24311533/42498598-2a5e9226-845f-11e8-9f82-4a36f0070157.png)
I fixed expokit to 2.5.10 but there are other errors. I searched about the error, your suggestion is to fix FBAudience to 4.28.1 in podlike.lock. But I failed to do pod install.
![screen shot 2018-07-10 at 4 36 43 pm](https://user-images.githubusercontent.com/24311533/42498700-7539a808-845f-11e8-9524-f0c06af2d500.png)
